### PR TITLE
Adding MVVM Light / Ninject Project

### PR DESCRIPTION
**Pull request by gcaufield**
There is still more work needed on MVVM Views but we can pull in the project updates

### CODE REVIEW

**Comment by StasTserk** [comment]
I'm not a fan of the project name that all of the common stuff went into. "Data" seems a misnomer for something that contains repositories, models, and controllers. Either the controller should be moved out or the name should be something descriptive imo.

File: MVVMViews/ViewModel/SimpleSpellListViewModel.cs [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+{
+    /// <summary>
+    /// This class contains properties that a View can data bind to.
+    /// <para>
+    /// See http://www.galasoft.ch/mvvm
+    /// </para>
+    /// </summary>
+    public class SimpleSpellListViewModel : ViewModelBase
+    {
+        private readonly ISpellLoader _loader;
```
Comment (StasTserk) [inline MVVMViews/ViewModel/SimpleSpellListViewModel.cs]: I think the correct thing to use here is the ISpellListController which would provide more high level functionality. ISpellLoader is the lower level repository that just brings the data into the app.

Comment (gcaufield) [inline MVVMViews/ViewModel/SimpleSpellListViewModel.cs]: Agreed

**Comment by gcaufield** [comment]
I moved the controller into a Services project

**Pull request merged**